### PR TITLE
PEP8 (style) changes only

### PR DESCRIPTION
**Pull request by nicholascar**
Manual and automated edits from the flak8 and black packages to modernise the style of code in this repo. No functional changes.

### CODE REVIEW

File: SPARQLWrapper/SPARQLExceptions.py [hunk 1]
```diff
@@ -46,23 +49,25 @@ class EndPointInternalError(SPARQLWrapperException):
...
+    msg = "The endpoint returned the HTTP status code 500"
 
 
 class QueryBadFormed(SPARQLWrapperException):
     """
     Query Bad Formed exception. Usually HTTP response status code ``400``.
     """
 
-    msg = "a bad request has been sent to the endpoint, probably the sparql query is bad formed"
+    msg = "A bad request has been sent to the endpoint: it is probably the SPARQL query is badly formed"
```
Comment (graue70) [inline SPARQLWrapper/SPARQLExceptions.py:60]: This isn't proper English, is it?

File: SPARQLWrapper/SPARQLExceptions.py [hunk 2]
```diff
@@ -46,23 +49,25 @@ class EndPointInternalError(SPARQLWrapperException):
...
 
 
 class EndPointNotFound(SPARQLWrapperException):
     """
     End Point Not Found exception. Usually HTTP response status code ``404``.
     """
 
-    msg = "it was impossible to connect with the endpoint in that address, check if it is correct"
+    msg = (
+        "It was not possible to connect with the endpoint given, check if it is correct"
```
Comment (graue70) [inline SPARQLWrapper/SPARQLExceptions.py:69]: given endpoint?

File: SPARQLWrapper/SPARQLExceptions.py (hunk 1 above)
Comment (white-gecko) [reply SPARQLWrapper/SPARQLExceptions.py]: should be: "A bad request has been sent to the endpoint: probably the SPARQL query is badly formed"

File: SPARQLWrapper/SPARQLExceptions.py (hunk 2 above)
Comment (white-gecko) [reply SPARQLWrapper/SPARQLExceptions.py]: sounds good, I would maybe write "It was not possible to connect to the endpoint given, check if it is correct"

File: test/stardog__lindas__test.py [hunk 3]
```diff
@@ -878,7 +909,7 @@ def testConstructByGETinCSV_Unexpected(self):
         ct = result.info()["content-type"]
         assert True in [one in ct for one in _SPARQL_DESCRIBE_CONSTRUCT_POSSIBLE]
         results = result.convert()
-        self.assertEqual(type(results), bytes) # text/turtle
+        self.assertEqual(type(results), bytes)  # text/turtle
```
Comment (white-gecko) [inline test/stardog__lindas__test.py:912]: Why do you add an extra space here?

**Review by white-gecko — APPROVED** [review]
In general it looks ok to me.

File: test/stardog__lindas__test.py (hunk 3 above)
Comment (nicholascar) [reply test/stardog__lindas__test.py:912]: It's one of the PEP conventions, so automatically added by Black

**Pull request merged**